src/client/app/chat/chat.component.html

```diff
@@ -253,7 +253,7 @@ <h3 class="card-title">{{doctor.name}}</h3>
             <!-- end chat-history -->
 
             <div class="container">
-                <form novalidate [formGroup]="message" (keydown.enter)="sendMessage(message)" (ngSubmit)="sendMessage(message)">
+                <form novalidate [formGroup]="message" (keydown.enter)="sendMessage(message, $event)" (ngSubmit)="sendMessage(message)">
```

Comment:
It will create errors because we are not emitting event during all invocations to `sendMessage()`.